ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse.yaml

```diff
@@ -33,13 +33,20 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: installer-rehearse-aws
+- as: installer-rehearse-aws-ipi
   interval: 960h
   steps:
     cluster_profile: aws-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
-    workflow: cucushift-installer-rehearse-aws-ipi-byo-route53
+    workflow: cucushift-installer-rehearse-aws-ipi
+- as: installer-rehearse-aws-ipi-disconnected-private-usgov
```

Comment:
suggest to naming it with cluster profile name, e.g: `installer-rehearse-aws-usgov-qe`